src/test/globalSetup.ts

```diff
@@ -1,19 +1,7 @@
 // Jest expects a single default function to be exported from this file.
 /* eslint-disable import/no-default-export */
-import dotenv from 'dotenv';
 import type { Config } from 'jest';
 
-// TODO: reconsider use of '.env.test' if or when
-// https://github.com/ThomWright/postgres-migrations/pull/93 gets merged/fixed.
-dotenv.config({ path: '.env.test' });
```

Comment:
Oh - this seems relevant! Currently my `.env.test` is pointing at a different database than my `.env`. Is that still necessary? In #611 we switched to a fork of `postgres-migrations` which fixes the schema issue, and it looks like that PR starts making use of the new functionality. Is it safe to point the tests at the same database as my local dev instance?

Reply:
@jasonaowen yes indeed you can use the local dev instance!